server/data/accreditedProgrammesApi/referralClient.test.ts

```diff
@@ -68,20 +65,19 @@ pactWith({ consumer: 'Accredited Programmes UI', provider: 'Accredited Programme
             authorization: `Bearer ${userToken}`,
           },
           method: 'GET',
-          path: apiPaths.referrals.show({ referralId: referral.id }),
+          path: apiPaths.referrals.show({ referralId: '0c46ed09-170b-4c0f-aee8-a24eeaeeddaa' }),
         },
       })
     })
 
     it('fetches the given referral', async () => {
-      const result = await referralClient.find(referral.id)
+      const result = await referralClient.find('0c46ed09-170b-4c0f-aee8-a24eeaeeddaa')
 
       expect(result).toEqual(referral)
     })
   })
 
   describe('findReferralSummaries', () => {
-    const organisationId = 'a026cc07-8e0b-40dd-9b66-1b3dacecc63d'
     const paginatedReferralSummaries: Paginated<ReferralSummary> = {
       content: referralSummaryFactory.buildList(2),
```

Comment:
Is it worth popping in the `status: REFERRAL_SUBMITTED` here to make it a bit clearer that these are the referral summaries we're looking for?

Reply:
Technically that's only in the condition with query parameters, I think? But I could add that in that condition

Reply:
I think it might make it clearer, but not too precious about it